databricks/koalas/utils.py

```diff
@@ -813,3 +813,31 @@ def compare_disallow_null(left, right, comp):
 
 def compare_allow_null(left, right, comp):
     return left.isNull() | right.isNull() | comp(left, right)
+
+
+def check_same_length(left: "IndexOpsMixin", right: Union[list, tuple]):
```

Comment:
Nice utility! The function name might be misleading while concluding its return type. Would it be possible to annotate the return type?